Add description of project into README.md

Loreaylen opened the pull request:
Added project description and features. Add and explanation of each view, color palette, fonts and so on.

Loreaylen committed:
Add description of project into README.md

Added project description and features. Add and explanation of each view, color palette, fonts and so on.

Luvwen reviewed (commented):
Agregar algún título, subtitulo y hacer una lista con las funcionalidades principales para no tener tanto texto junto. De resto todo bien! 🚀

Luvwen reviewed (approved):
Revisar comentario 👍

valeriazacariasl reviewed (commented):
👌

Loreaylen committed:
Update description of project in README.md

Updated README.md to add a better description of current features. Added  titles and subtitles.
